### PR TITLE
Misspelling

### PR DESCRIPTION
**Pull request by fmrtn**
Is this the right way for PR?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ This will be the repo to help to a friend of mine to learn something more the gi
 
 ## Content
 
-Jus push a new line to generate another commit in the repository.
+Just push a new line to generate another commit in the repository.
```
Comment (josesalvador) [inline README.md:10]: Good catch!!

File: file [hunk 2]
```diff
@@ -0,0 +1 @@
+This is an empty file
```
Comment (josesalvador) [inline file]: Don't you mind to set a proper name and content to file?

Something with some sense, please, not a _file_ as filename and without any type of content.

**Review by josesalvador — CHANGES_REQUESTED** [review]
Good work.

I left a minor question for you that you have to reolsve in order I can approve your PR.

Take a llok, please.

**Comment by fmrtn** [comment]
Done.

File: file (hunk 2 above)
Comment (fmrtn) [reply file]: I've change the filename.

**Review by josesalvador — APPROVED** [review]
Good job!!

**Pull request merged**